src/screens/ParadeMapScreen/Map.js

```diff
@@ -189,6 +187,13 @@ class Map extends PureComponent<Props, State> {
 
     return (
       <View style={styles.mapWrapper}>
+        {this.state.userLocation.type === "authorized" &&
+          this.state.userLocation.location.type === "tracking" && (
```

Comment:
I believe this should be `error` rather than `tracking`. Currently this message is displaying if we can find the users location.

Reply:
One minor bug that I can see happening, is if the state of userLocation changes out of the authorized error state before the MessageBanner animates out, it will disappear (without animating). One potential solution is to always render the MessageBanner and pass an enabled prop. Then use `getDerivedStateFromProps` to calculate state on the enabled prop change.